guide/private-key-management/multi-user-schemes.md

```diff
@@ -52,38 +52,41 @@ Just like with personal schemes, external signing devices can be used to hold on
 ### How it works
 A wallet or coordination application initiates a multi-sig wallet, choosing the number of total keys, and the number required to sign transactions. It then needs to add private keys from the other user's wallet. It can then complete the creation process. For any future transaction from the multi-sig wallet the required amount of co-signers need to sign (using Partially Signed Bitcoin Transactions - PSBT from [BIP174]({{ 'https://github.com/bitcoin/bips/blob/master/bip-0174.mediawiki'}}){:target="_blank"}) before any transaction is valid.
 
-#### Pros
+{% include fact/pros.html %}
+
 - Allows several people to access and control a bitcoin wallet
 - Can tailor requirements for multiple co-signers and access
 - High security, if setup well
 
-#### Cons
+{% include fact/close.html %}
+
+{% include fact/cons.html %}
+
 - Has significant complexity and op-sec burden for multiple private keys, each of which might need a good manual backup scheme
 
+{% include fact/close.html %}
+
 ### Best practice
 
-**When to use**
+#### When to use
 - When funds need to be accessed by several people or an organization
 - When users are likely to be very knowledgeable or be guided through setup and use
 
-**When not to use**
+#### When not to use
 - (No other option currently for shared access scenarios)
 
-**Variations**
+#### Variations
 - The number of required co-signers and attached keypairs
 - Key storage devices
 
-**Do's**
+#### Do's
 - Make sure the multi-key setup itself is backed up properly, including x-pubs for all the participating keys, fingerprint and derivation.
 
-**Products that use this scheme**
-Few tailor-made products exist for shared wallets, but any wallet application that supports multi-key setups can be used to initiate a shared wallet.
-- [Electrum](https://electrum.org){:target="_blank"}
-- [Bluewallet](https://bluewallet.io){:target="_blank"}
-- [Specter](https://specter.solutions){:target="_blank"}
-- [Armory](https://btcarmory.com){:target="_blank"}
-- [Guarda](https://guarda.com){:target="_blank"}
-- [Revault](https://revault.dev){:target="_blank"} - in development
+{% include fact/open.html color="blue" label="Products that use this scheme" icon="rectangle" %}
+
+[Electrum](https://electrum.org){:target="_blank"}, [Bluewallet](https://bluewallet.io){:target="_blank"}, [Specter](https://specter.solutions){:target="_blank"}, [Armory](https://btcarmory.com){:target="_blank"},[Nunchuk](https://nunchuk.io/){:target="_blank"}, [Sparrow](https://sparrowwallet.com/){:target="_blank"}, [Guarda](https://guarda.com){:target="_blank"}, [Revault](https://revault.dev){:target="_blank"}.
```

Comment:
```suggestion
[Electrum](https://electrum.org), [Bluewallet](https://bluewallet.io), [Specter](https://specter.solutions), [Armory](https://btcarmory.com),[Nunchuk](https://nunchuk.io/), [Sparrow](https://sparrowwallet.com/), [Guarda](https://guarda.com), [Revault](https://revault.dev).
```
We have a plug-in now that automatically adds "target=blank" to external links.

Reply:
They were just copied and pasted from the old content - will update this today